### PR TITLE
documentation: Added README Fix #84

### LINKED ISSUE #84: General README
_Migrated from JIRA [RUCIO-2858](https://its.cern.ch/jira/browse/RUCIO-2858)_

### PR DESCRIPTION
**Pull request by vingar**

### CODE REVIEW

File: README.client.rst [hunk 1]
```diff
@@ -1,9 +1,35 @@
-Rucio Client Lite Package
+Rucio: Client Lite Package
```
Comment (bari12) [inline README.client.rst]: The Package names are inconsistent
> Client: Rucio: Client Lite Package
> Main: Rucio: Data Management for science in the Big Data era
> WebUI: Rucio WebUI Package

I think this should be something consistent there. Rucio Clients / Rucio / Rucio WebUI (No package)

File: README.client.rst [hunk 2]
```diff
@@ -1,9 +1,35 @@
...
 =========================
 
+Rucio is a project that provides services and associated libraries for allowing scientific
```
Comment (bari12) [inline README.client.rst]: I like having this sentence in every README, we should just wait for the confirmed one on Tuesday.

File: README.client.rst [hunk 3]
```diff
@@ -1,9 +1,35 @@
...
 
-See release notes and more at `<https://its.cern.ch/jira/browse/RUCIO>`_.
+General information and latest documentation about Rucio can be found
+at http://rucio.readthedocs.io.
+
+Developers
+----------
+
+For information on how to contribute to Rucio, please refer and follow our
+`Guidelines <http://rucio.readthedocs.io/contributing.html>`_ or CONTRIBUTING_.
```
Comment (bari12) [inline README.client.rst]: I would only link to CONTRIBUTING_ here. In the CONTRIBUTING then we can have more links.

**Comment by bari12** [comment]
In my opinion it is good like this. We can either wait for Tuesday for the final "sentence" or just put it now and update it on Tuesday. Opinions?

**Review by mlassnig — APPROVED** [review]
I think it's good like this. If we get an updated sentence on Tuesday we can always update.

**Pull request merged**